tools/releases/dockerfiles/Dockerfile.kuma-cp

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
```

Comment:
Docker image should run under a non-`root` account, e.g. `nobody` (`65534:65534`) on `alpine`